src/sentry/attachments/redis.py

```diff
@@ -0,0 +1,18 @@
+"""
+sentry.attachments.redis
+~~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2010-2014 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+
+from __future__ import absolute_import
+
+from sentry.cache.redis import RedisCache
+
+from .base import BaseAttachmentCache
+
+
+class RedisAttachmentCache(BaseAttachmentCache):
+    def __init__(self, **options):
+        super(RedisAttachmentCache, self).__init__(RedisCache(**options), **options)
```

Comment:
This isn't going to allow us to use Redis Cluster since `RedisCache` doesn't. But I'm not sure if that's still the plan or not. I'd hope it is so we get HA for free.

@tonyo @JTCunning

Reply:
@mattrobenolt the plan is to have a real Redis cluster ready tomorrow.

Reply:
I mean, is it actually using Redis Cluster? I'm being very explicitly with my words here. If so, this usage of `RedisCache` won't work since it doesn't use the RedisCluster library.

Reply:
What Matt is referring to is that we need to use a Redis client that is compatible with Redis Cluster. This is the current client we have written: https://github.com/getsentry/sentry/blob/master/src/sentry/utils/redis.py#L79

Reply:
And for reference, this is how we need to use it: https://github.com/getsentry/sentry/blob/master/src/sentry/similarity/__init__.py#L74

Something similar.

Reply:
Perfect, I’ll update the PR and will ping you when ready.